tests/Jwt/FirebaseParserTest.php

```diff
@@ -53,30 +53,12 @@ protected function setUp()
 
     public function testParseTokenWithExpiredToken()
     {
-        $parser = new FirebaseParser($this->config);
-
-        try {
-            $parser->parseToken(
-                'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ1cmkiLCJpYXQiOiIxNDQzNzE5MjM2IiwiZXhwIjoxNDQzNzIyODM2LCJzdWIiOiJ1c2VyLTEifQ.YquvWRtH72muTv1hCCKH4HBkBhhCm1wTG0MpnyC-pvQ'
-            );
-            $this->fail('Expected exception was not thrown');
-        } catch (InvalidException $e) {
-            $this->assertSame(InvalidException::CODE_TOKEN_EXPIRED, $e->getCode());
-        }
-    }
+        $this->setExpectedException(InvalidException::class);
```

Comment:
Or rather, add the `$code` parameter to this call? 

https://github.com/sebastianbergmann/phpunit/blob/202600758fa4794ae83b178b919756f69eb2ce59/src/Framework/TestCase.php#L467